applications/EigenSolversApplication/custom_solvers/eigen_pardiso_ldlt_solver.h

```diff
@@ -0,0 +1,75 @@
+/*
+//  KRATOS _______
+//        / ____(_)___ ____  ____
+//       / __/ / / __ `/ _ \/ __ \
+//      / /___/ / /_/ /  __/ / / /
+//     /_____/_/\__, /\___/_/ /_/ SolversApplication
+//             /____/
+//
+//  Author: Thomas Oberbichler
+*/
+
+#if !defined(KRATOS_EIGEN_PARDISO_LDLT_SOLVER_H_INCLUDED)
+#define KRATOS_EIGEN_PARDISO_LDLT_SOLVER_H_INCLUDED
+
+// External includes
+#include <Eigen/Sparse>
+#include <Eigen/PardisoSupport>
+
+// Project includes
+#include "includes/define.h"
+#include "linear_solvers/direct_solver.h"
+#include "spaces/ublas_space.h"
+#include "includes/ublas_interface.h"
+#include "includes/ublas_complex_interface.h"
+
+namespace Kratos {
+
+template <typename TScalar = double>
+class EigenPardisoLDLTSolver
+{
+public:
+    using Scalar = TScalar;
+    using SparseMatrix = Eigen::SparseMatrix<Scalar, Eigen::RowMajor, int>;
+    using Vector = Eigen::Matrix<Scalar, Eigen::Dynamic, 1>;
+
+private:
+    Eigen::PardisoLDLT<SparseMatrix> m_solver;
```

Comment:
as far as i understand there are two layers now?
* EigenDirectSolver
  * EigenPardisoLDLTSolver
    * Eigen::PardisoLDLT
and the childs are always called m_solver?

Reply:
It is the same as before. The last "layer" was just hidden by the templating